config/config.go

```diff
@@ -239,6 +250,47 @@ func (t *TargetConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return checkOverflow(t.XXX, "target")
 }
 
+//
+//AWS Secret
+//
```

Comment:
As per the `gofmt -d` output:

```suggestion
//AWS Secret
```

Reply:
I think it's still not enough. We also need to remove heading and trailing slashes. Just like in the suggestion I made above. 🙂👍

Reply:
Thanks, will address that